packages/scandipwa/src/component/SwipeToDelete/SwipeToDelete.component.js

```diff
@@ -238,6 +242,7 @@ export class SwipeToDelete extends PureComponent {
               onDragEnd={ this.handleDragEnd }
               mix={ topElemMix }
             >
+                <Loader isLoading={ isLoading && isAheadRemoveItemThreshold } />
```

Comment:
Don't show loader if SwipeToDelete is not full swiped. It will prevent from double loader if remove item with button 'Delete'